### PR TITLE
Beacd-18: AEOI

### PR DESCRIPTION
**Pull request by prashantchoudhari**

### CODE REVIEW

**Comment by prashantchoudhari** [comment]
*********** template modification pending ***********

File: app/config/FrontendAppConfig.scala [hunk 1]
```diff
@@ -78,4 +81,12 @@ class FrontendAppConfig @Inject()(override val runModeConfiguration: Configurati
   lazy val publishedAssets = loadConfig(s"urls.external.assets.host")
   def getPublishedAssetsUrl(key: String): String = publishedAssets + loadConfig(s"urls.external.assets.$key")
 
+  def emacEnrollmentsUrl(enrolment: Enrolments): String = {
+    s"$enrolmentManagementFrontendHost/enrolment-management-frontend/${enrolment.toString}/request-access-tax-scheme?continue=%2Fbusiness-account"
```
Comment (kurtlogan) [inline app/config/FrontendAppConfig.scala]: Should we put this in the config?

Comment (prashantchoudhari) [reply app/config/FrontendAppConfig.scala]: yes, we should - changing. I just lifted as it is.

File: app/views/other/aeoi/haveYouRegisteredAEOI.scala.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+@import controllers.other.aeoi.routes._
+@import models.other.aeoi.HaveYouRegisteredAEOI
+
+
+@(appConfig: FrontendAppConfig, form: Form[_])(serviceInfoContent: Html)(implicit request: Request[_], messages: Messages)
+
+@main_template(
+    title = messages("haveYouRegisteredAEOI.title"),
+    appConfig = appConfig,
+bodyClasses = None,
```
Comment (kurtlogan) [inline app/views/other/aeoi/haveYouRegisteredAEOI.scala.html]: tab code in

Comment (prashantchoudhari) [reply app/views/other/aeoi/haveYouRegisteredAEOI.scala.html]: have done it.. I will push when I merge master last time after all PR's are merged.. thank you

**Pull request merged**